http-client-core/src/main/java/io/micronaut/http/client/annotation/Client.java

```diff
@@ -109,4 +132,38 @@ String[] alpnModes() default {
         HttpVersionSelection.ALPN_HTTP_2,
         HttpVersionSelection.ALPN_HTTP_1
     };
+
+    /**
+     * The interface definition type.
+     *
+     * @since 4.8.0
+     */
+    enum DefinitionType {
```

Comment:
can you annotate this with `@Experiemental`

Reply:
fixed. Sorry, again force pushed. Forgot